src/main/resources/de/theit/jenkins/crowd/CrowdSecurityRealm/CacheConfiguration/config.jelly

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+The MIT License
+
+Copyright 2014 Jesse Glick.
```

Comment:
LOL. A too quick copy/paste. I'll fix it.

Le samedi 4 avril 2015, Kanstantsin Shautsou <notifications@github.com> a
écrit :

> In
> src/main/resources/de/theit/jenkins/crowd/CrowdSecurityRealm/CacheConfiguration/config.jelly
> <https://github.com/jenkinsci/crowd2-plugin/pull/10#discussion_r27767473>:
>
> > @@ -0,0 +1,34 @@
> > +<?xml version="1.0" encoding="UTF-8"?>
> > +<!--
> > +The MIT License
> > +
> > +Copyright 2014 Jesse Glick.
>
> @jglick <https://github.com/jglick> really?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jenkinsci/crowd2-plugin/pull/10/files#r27767473>.
>


-- 
-----
Arnaud Héritier
http://aheritier.net
Mail/GTalk: aheritier AT gmail DOT com
Twitter/Skype : aheritier